DomainContent/Trivia/FourSquareTrivia/triviaColorCheckBlue.js

```diff
@@ -0,0 +1,116 @@
+// triviaColorCheck.js
+
+//  Created by Mark Brosche on 11/11/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+/* global EventBridge Users AccountServices */
+
+(function(){
+    var _this,
+        bubble,
+        gameOn,
+        gameZone;
+    
+    var DISQUALIFIED_POSITION = { x: -69.4697, y: -19, z: 6.3204 },
+        HALF_MULTIPLIER = 0.5,
+        ZONE_SQUARE_RADIUS = 1.5,
+        RADIUS = 50,
+        RANGE = 5;
+
+    var ColorCheck = function(){
+        _this = this;
+    };
+
+    ColorCheck.prototype = {
+        preload: function(entityID){
+            _this.entityID = entityID;
+            gameZone = Entities.getEntityProperties(
+                Entities.findEntitiesByName("Trivia Player Game Zone", MyAvatar.position, RADIUS)[0], ['position', 'rotation', 'dimensions']);
```

Comment:
line is longer than max of 128 char per line...also same for the other files like this

Reply:
fixed